docs/releases/2.2.txt

```diff
@@ -309,6 +309,13 @@ properly (the database was empty at the end of the whole test suite). This
 change shouldn't have an impact on your tests unless you've customized
 :class:`~django.test.TransactionTestCase`'s internals.
 
+``sqlparse`` is required dependency
+-----------------------------------
+
+To simplify a few parts of Django's database handling, `sqlparse
+<https://pypi.org/project/sqlparse/>`_ is now a required dependency. It's
+automatically installed along with Django.
```

Comment:
I see this wording was used for `pytz` in 1.11 but I'm not sure about saying "it's automatically installed" - it only is if you are downloading a tarball from PyPI. But I guess if no one complained for `pytz`, `¯\_(ツ)_/¯`

Reply:
I'm not sure of an install scenario where it wouldn't be installed?

Reply:
What I meant is some obscure installation method like unpackaging the tarball directly would not get it automatically. But I suppose for all the [documented installation methods](https://docs.djangoproject.com/en/2.1/topics/install/) it would be, so the wording is fine.

Reply:
Do you want to bring it up on the mailing list?